README.md

```diff
@@ -22,7 +22,7 @@ See also this simple usage example: https://github.com/lorensr/meteor-starter-ki
 
 ```
 meteor add apollo
-meteor npm install --save apollo-client apollo-server express
+meteor npm install --save apollo-client apollo-server express graphql graphql-tools
```

Comment:
I'm actually not sure why we force the users to add our dependencies.. is it because we cannot get meteor to combine package npm dependencies together with app package.json dependencies?